3rdparty/everest/library/Hacl_Curve25519_joined.c

```diff
@@ -18,6 +18,12 @@
  *
  *  This file is part of mbed TLS (https://tls.mbed.org)
  */
+#ifndef _BSD_SOURCE
+#define _BSD_SOURCE /* Required to enable compilation under --std=c99 */
+#endif
+#ifndef _DEFAULT_SOURCE
+#define _DEFAULT_SOURCE
```

Comment:
```suggestion
/* (modern version of _BSD_SOURCE) */
#define _DEFAULT_SOURCE
```

Reply:
See https://stackoverflow.com/a/29201732/7761803 (not clear that `_BSD_SOURCE` is still required, but let's continue to support older systems)